config/environments/production.rb

```diff
@@ -100,7 +97,4 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
-  PUBLIC_URL = ENV['APPLICATION_URL']
-
-  config.secret_key_base = ENV['SECRET_KEY_BASE']
```

Comment:
Standard for > Rails 4.2 to have it in `config/secrets.yml`.
Was defined two times.